src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -757,7 +752,7 @@ private protected override void SaveModel(ModelSaveContext ctx)
             var buffer = _session.graph.ToGraphDef(status);
             ctx.SaveBinaryStream("TFModel", w =>
             {
-                w.WriteByteArray(buffer.Data);
+                w.WriteByteArray(buffer.MemoryBlock.ToArray());
```

Comment:
>w.WriteByteArray(buffer.MemoryBlock.ToArray()); [](start = 16, length = 47)

can you please do a diff between the graph with your change and master and see if there are any changes? I know the graph size increased and I want to know why.